src/test/java/Page_Objects/MainMenuPage.java

```diff
@@ -28,6 +33,7 @@ public class MainMenuPage {
     public MainMenuPage(WebDriver driver) {
         this.driver = driver;
         PageFactory.initElements(driver, this);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));

```

Comment:
I need that `WebDriverWait` constructor for method called `toggleFullscreen`, where I have to wait for `visibilityOf(fullscreenElement)`, because of the `StaleElementReferenceException`. But I moved that line to the method already.